de.fu_berlin.inf.dpp/build.gradle

```diff
@@ -0,0 +1,176 @@
+buildscript {
+  repositories {
+    maven {
+      url  "http://dl.bintray.com/monkey-works/BuildMonkey"
+    }
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
+ dependencies {
+    classpath "de.monkeyworks.buildmonkey:gradle.dependency:0.4.13"
+  }
+}
+
+apply plugin: 'de.monkeyworks.buildmonkey.ManifestDependencyPlugin'
```

Comment:
Does this also check the class access restriction that is part of the OSGi specification ?

E.g

You can convert an Eclipse IWorkspace object to an actual Eclipse Workspace object. However you will get an exception because the OSGi classloader will not load the Workspace class.